bgpd/bgp_route.c

```diff
@@ -4487,7 +4487,8 @@ void bgp_update(struct peer *peer, const struct prefix *p, uint32_t addpath_id,
 		/* Update MPLS label */
 		if (has_valid_label) {
 			extra = bgp_path_info_extra_get(pi);
-			if (extra->label != label) {
+			if (memcmp(&extra->label[0], &label[0],
+				   num_labels * sizeof(mpls_label_t))) {
```

Comment:
"bgpd: clarify pointer comparison" commit


```
Fixes: https://github.com/FRRouting/frr/commit/8ba7105057357c566a0475b32829241cca85d848 ("bgpd: fix valgrind flagged errors")

Signed-off-by: Philippe Guibert <philippe.guibert@6wind.com>
```
No new line between Fixes and Signed-off-by

it is not clarify but fix. Say in title which pointer we are talking about